torch/_torch_docs.py

```diff
@@ -3571,6 +3571,30 @@ def merge_dicts(*dicts):
     tensor([15, 30, 15])
 """.format(**common_args))
 
+add_docstr(torch.ldexp, r"""
+ldexp(input, other, *, out=None) -> Tensor
+
+Computes :math`\text{{input}} * 2^\text{{other}}`, element-wise.
+
+The mantissas in :attr:`input` are multipled by a specified integral power of
```

Comment:
A one-sentence description like this should appear first. This description is good but it assumes a particular mathematical perspective (that the value in input represent mantissas). What about providing a perspective-free one sentence description, the mathematical formula, and then a sentence describing the function's intended or typical use case? For example:

"Multiplies input by 2**other. 

<formula with _i notation.

Typically...
"

See torch.sub for a good way to write the mathematical formula.

Aside: "multipled" -> "multiplied"